src/main/java/com/mraof/minestuck/data/loot_table/MSBlockLootTables.java

```diff
@@ -285,6 +287,29 @@ protected void addTables()
 		dropSelf(DEAD_PLANKS_SLAB);
 		dropSelf(TREATED_PLANKS_SLAB);
 		
+		dropSelf(TRAJECTORY_BLOCK);
+		dropSelf(STAT_STORER);
+		dropSelf(REMOTE_OBSERVER);
+		dropSelf(WIRELESS_REDSTONE_TRANSMITTER);
+		dropSelf(WIRELESS_REDSTONE_RECEIVER);
+		dropSelf(SOLID_SWITCH);
+		dropSelf(VARIABLE_SOLID_SWITCH);
+		dropSelf(ONE_SECOND_INTERVAL_TIMED_SOLID_SWITCH);
+		dropSelf(TWO_SECOND_INTERVAL_TIMED_SOLID_SWITCH);
+		add(SUMMONER, noDrop());
+		add(AREA_EFFECT_BLOCK, noDrop());
+		dropSelf(PLATFORM_GENERATOR);
+		add(PLATFORM_BLOCK, noDrop());
+		dropSelf(ITEM_MAGNET);
+		dropSelf(REDSTONE_CLOCK);
+		dropSelf(FALL_PAD);
+		dropSelf(AND_GATE_BLOCK);
+		dropSelf(OR_GATE_BLOCK);
+		dropSelf(XOR_GATE_BLOCK);
+		dropSelf(NAND_GATE_BLOCK);
+		dropSelf(NOR_GATE_BLOCK);
+		dropSelf(XNOR_GATE_BLOCK);
```

Comment:
I intend to leave them in a position similar to command blocks, where they can be held in the inventory(and picked up, unlike command blocks) and used like normal but do not appear in creative tabs or jei

Reply:
If you're making it possible to pick them up (and I assume in a future PR make them generate in the world, thus making it possible to obtain them during normal play), why would you not also let them show up in jei/creative tabs? I'd kinda expect them to be readily available if they'll be available during normal play in survival.

Reply:
While they are available to collect and use in survival, less attention is intended to be brought to them because they do not match the aesthetics/functionality common to the mod

Reply:
I've still got doubts about excluding them from the creative inventory just for that reason. Gonna think about it some more and see if I can formulate something more properly.